GVFS/GVFS.Installer.Mac/scripts/postinstall

```diff
@@ -1,4 +1,29 @@
 #!/bin/bash
+
+# Load PrjFSKext if it is not loaded already
+# PrjFSKext is an IOKit kext and should get automatically loaded 
+# by macOS after install. But the system does not seem to auto-load it
+# consistently after every install. The code below checks if it has
+# gotten auto-loaded. If not load it.
+# The BundleID of the Kext(ProjFS.Mac/PrjFS.xcodeproj) is defined in
+# ProjFS.Mac/PrjFS.xcodeproj project.
+KEXTBUNDLEID="org.vfsforgit.PrjFSKext"
+KEXTPATH="/Library/Extensions/PrjFSKext.kext"
+kextstatCmd="/usr/sbin/kextstat -l -b $KEXTBUNDLEID"
+kextstatOutput=$(eval $kextstatCmd)
+if [[ -z "${kextstatOutput// }" ]]; then
+# load the kext using kextload command. Installer is run as the
+# admin user, so we already have required privileges. In case load
+# still fails, then exit 1. This will cause the installer to display
+# an installation failed error message.
+    loadCmd="/sbin/kextload \"$KEXTPATH\""
+    echo $loadCmd
+    eval $loadCmd || exit 1
+else
+    echo "$kextstatCmd returned non-zero output. This might possibly indicate an error."
```

Comment:
When wouldn't this be an error? What action do we expect a user to take here? This would be a very frustrating error message to see because I would walk away feeling helpless.

Reply:
>> When wouldn't this be an error?

Two scenarios here -
1. If macOS successfully auto-loaded the kext before this script(`postinstall`), then this will not be an error. This would be the best case scenario. User need to take no action here. The output of `echo` goes to the installer log file only, NO install error will get displayed to the user. The installer log would be useful to us developers (in case of issues like - installation succeeded but still `gvfs mount` command prompts for `kextload`).
2. `kextstat` itself fails with error output. I haven't seen this happen, but `man kextstat` does say kextstat can possibly exit with error. In this case, again, user will not see any error in installer UI. `gvfs mount` will prompt user to manually do a `kextload`. `GVFS.Service` auto-mount will fail, with error messaging in `Service` logs.

>> What action do we expect a user to take here? 

None. The two lines of logs are there to help us developers debug kext load issue (in the 2nd case above) during install. 

>> This would be a very frustrating error message to see because I would walk away feeling helpless.

Agree. Installer displays no error. The two lines go to installer log only.